website/screens/components/resultset-table/code/ResultsetTableCodePage.tsx

```diff
@@ -81,12 +81,12 @@ const sections = [
             </td>
             <td>
               <TableCode>
-                {"{ displayValue: React.ReactNode; sortValue?: string; }[]"}
+                {"{ displayValue: React.ReactNode; sortValue?: string; }[][]"}
```

Comment:
While the typing is correct it is not very visual, maybe we can make better use of the description to improve this.
We can type it as Row[] and explain int the description:

`{ displayValue: React.ReactNode; sortValue?: string; } = Cell`
`Cell[] = Row`

Reply:
Sounds like a good idea, can you take a look at it now?